app/helpers/uploads_helper.rb

```diff
@@ -119,7 +119,7 @@ def upload_processing?(rec)
   end
 
   def upload_problem?(rec)
-    %w[invalid error].include?(rec&.status) || upload_stalled?(rec)
+    %w[invalid error].include?(rec&.status)
```

Comment:
I was putting the "error" badge on things that looked stalled (had been processing for > 1 minute or something).

Turns out, that tends to flag episodes that are slicing audio (the slice jobs often take longer).  Which was kind of annoying in the UI.  This shows the "processing" badge instead now.